joern-cli/frontends/csharpsrc2cpg/src/main/scala/io/joern/csharpsrc2cpg/astcreation/AstForStatementsCreator.scala

```diff
@@ -300,10 +300,12 @@ trait AstForStatementsCreator(implicit withSchemaValidation: ValidationMode) { t
       .code(code(usingStmt))
       .lineNumber(line(usingStmt))
       .columnNumber(column(usingStmt))
+    val declAst = Try(createDotNetNodeInfo(usingStmt.json(ParserKeys.Declaration))) match
+      case Success(declNodevalue) => astForNode(declNodevalue)
```

Comment:
you are missing `{}` after match. Though it is not a compulsion by the compiler. We should use the curly braces to make the code more readable.

Reply:
yes done, thanks